packages/commonwealth/package.json

```diff
@@ -183,6 +183,7 @@
     "express-validator": "^6.14.2",
     "feed": "^4.2.2",
     "firebase": "^10.12.1",
```

Comment:
I will approve but you should first test on staging. I have a feeling it will fail because you need to move the react dependency from `devDependencies` -> `dependencies`